addons/web/static/src/views/fields/field.js

```diff
@@ -14,6 +14,37 @@ const isSmall = utils.isSmall;
 const viewRegistry = registry.category("views");
 const fieldRegistry = registry.category("fields");
 
+const supportedFieldsTypes = [
+    // fields_binary.py
```

Comment:
I would remove the comments. It's not really useful, and likely to become outdated at some point